source/slang/hlsl.meta.slang

```diff
@@ -872,14 +872,16 @@ extension __TextureImpl<T,Shape,isArray,isMS,sampleCount,access,isShadow,0,forma
 
     [__readNone]
     [ForceInline]
-    [require(glsl_hlsl_spirv, texture_querylod)]
+    [require(glsl_hlsl_metal_spirv, texture_querylod)]
     float CalculateLevelOfDetail(SamplerState s, TextureCoord location)
     {
         __requireComputeDerivative();
         __target_switch
         {
         case hlsl:
-            __intrinsic_asm "CalculateLevelOfDetail";
+            __intrinsic_asm ".CalculateLevelOfDetail";
+        case metal:
+            __intrinsic_asm "$0.calculate_clamped_lod($1, $2)";
```

Comment:
Can be simplified to ".calculate_clamp_lod" so it is consistent with the HLSL case.

Reply:
Good catch. I will make the change and check if there are more like this